### PR TITLE
Fix minor typos in README and examples/

### PR DESCRIPTION
**Pull request by frapac**

### CODE REVIEW

File: examples/benders_quatile_regression.jl [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-#' Note that this is NOT the standard technique to solve Norm-1
-#' regressions. Taylor made methods are available
-#' [here](https://cran.r-project.org/web/packages/quantreg/quantreg.pdf)
-#' for instance.
-#'
-
-#' This notebook will require the following libraries:
-
-#' ParameterJuMP itself
+# ---
```
Comment (blegat) [inline examples/benders_quatile_regression.jl]: I think the `'` are on purpose because @joaquimg first transform it with Weave.

Comment (frapac) [reply examples/benders_quatile_regression.jl]: My bad... I should take a look on the Weave package.

**Comment by blegat** [comment]
@frapac Needs rebasing :)

**Comment by frapac** [comment]
Arf, I get some trouble with CTRLF and git...

File: examples/benders_quatile_regression.jl [hunk 2]
```diff
@@ -1,642 +1,642 @@
...
-#' JuMP: the julia mathematical programming modeling tool
-using JuMP
-
-#' GLPK: A linear programing solver (other solvers could be used - 
-#' such as Clp, Xpress, Gurobi, CPLEX and so on)
-
-#+ echo = false
-# using Xpress
-# const OPTIMIZER = Xpress.Optimizer;
+#' ---
```
Comment (blegat) [inline examples/benders_quatile_regression.jl]: What is your text editor ? Is is an issue between `\n` vs `\n\r` ?

Comment (frapac) [reply examples/benders_quatile_regression.jl]: Vim. But I think the issue is rather with git's `autocrlf` params. Fixing that ASAP!

**Review by joaquimg — APPROVED** [review]

**Pull request merged**